acceptance/framework/k8s/deploy.go

```diff
@@ -123,6 +160,38 @@ func CheckStaticServerConnectionMultipleFailureMessages(t *testing.T, options *k
 		}
 	})
 }
+func CheckStaticServerConnectionMultipleFailureMessagesJob(t *testing.T, options *k8s.KubectlOptions, sourceApp string, expectSuccess bool, failureMessages []string, expectedSuccessOutput string, curlArgs ...string) {
```

Comment:
Not needed if you use a varadic TestConnectionSuccess and pass the service through but you would need to change `CheckStaticServerConnectionMultipleFailureMessages` to determine if the sourceApp was the job.

Something like:

```
	resourceType := "deploy"
	if sourceApp == "job-client" {
		resourceType = "jobs"
	}

	args := []string{"exec", resourceType + sourceApp, "-c", sourceApp, "--", "curl", "-vvvsSf"}

```